conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,96 @@
+# Research Software Engineering Conference 2019
+
+* Conference website: https://rse.ac.uk/conf2019/
+* Application form: https://docs.google.com/forms/d/e/1FAIpQLScVml19n44QfuOfjuaRlJUtXNMih-X4xUrNjlr8XJVHeyTyQg/viewform
+* Location: University of Birmingham
+* Date: 17th - 19th September 2019
+
+### Workshop Length
+
+180 mins (to allow time for Kubernetes cluster deployment and an introduction to reproducible research and BinderHub).
+
+### Title
+
+"Build a BinderHub for hosting Reproducible Software in the Cloud"
+
+### Authors
+
+* [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson), ([The Alan Turing Institute](https://www.turing.ac.uk/)) on behalf of **[The Turing Way](https://github.com/alan-turing-institute/the-turing-way) Collaboration**.
+
+### Abstract
+
+_(1250 characters max, current count: 1244 [please update if you edit])_
+
+BinderHub is a Cloud-based, multi-user server technology that captures the computing environment of a GitHub repository within a Docker image.
```

Comment:
I feel like the capital "C" in Cloud looks weird.....

but I'm fine with keeping it in if you like it that way 😸

Reply:
I guess in my head, "the Cloud" is a proper noun 😂